tests/selenium/test-runner.js

```diff
@@ -1,8 +1,22 @@
+var server = require('../../server');
```

Comment:
test-runner now handles server, driver, and By...

It also handles calling next, and setting up and taking down driver and By.